onedal/basic_statistics/tests/test_basic_statistics.py

```diff
@@ -32,6 +32,14 @@
         ("standard_deviation", np.std, (3e-5, 3e-5)),
     ]
 
+    options_and_tests_csr = [
+        ("sum", "sum", (1e-5, 1e-7)),
+        ("min", "min", (1e-5, 1e-7)),
+        # There is a bug in oneDAL's max computations on GPU
+        #         ("max", "max", (1e-5, 1e-7)),
```

Comment:
Please create ticket for this if there is no any

Reply:
Done